### ISSUE TITLE
Anaconda Navigator not opening after installing fbprophet

### ISSUE THREAD

**Issue by premkilaru** [ContinuumIO/anaconda-issues#8981]
I have installed fbprophet using 

conda install -c conda-forge fbprophet

After that I am not able to open Anaconda Navigator 

I tried opening Jupyter Notebook directly then  I typed below import command

from fbprophet import Prophet

I got the below error.
-----------Error Begin-----------------------------------------------------
PermissionError                           Traceback (most recent call last)
<ipython-input-2-3dac74d0cfd4> in <module>()
----> 1 from fbprophet import Prophet

C:\ProgramData\Anaconda3\lib\site-packages\fbprophet\__init__.py in <module>()
      6 # of patent rights can be found in the PATENTS file in the same directory.
      7 
----> 8 from fbprophet.forecaster import Prophet
      9 
     10 __version__ = '0.2.1'

C:\ProgramData\Anaconda3\lib\site-packages\fbprophet\forecaster.py in <module>()
     26 
     27 # fb-block 1 start
---> 28 from fbprophet.models import prophet_stan_models
     29 # fb-block 1 end
     30 

C:\ProgramData\Anaconda3\lib\site-packages\fbprophet\models.py in <module>()
     33 
     34 prophet_stan_models = {
---> 35     'linear': get_prophet_stan_model('linear'),
     36     'logistic': get_prophet_stan_model('logistic'),
     37 }

C:\ProgramData\Anaconda3\lib\site-packages\fbprophet\models.py in get_prophet_stan_model(model)
     28     )
     29     # fb-block 4 end
---> 30     with open(model_file, 'rb') as f:
     31         return pickle.load(f)
     32 

PermissionError: [Errno 13] Permission denied: 'C:\\ProgramData\\Anaconda3\\lib\\site-packages\\fbprophet\\stan_models\\linear_growth.pkl'

-----------Error End-----------------------------------------------------

Thank you for the help, happy to provide more information as needed.

**Comment by goanpeca** [comment]
Please provide the output of

```
conda info
```

and

```
conda list --show-channel-urls
```

**Comment by premkilaru** [comment]
Hello goanpeca, 

Thank you for the response. Since I did this in my work machine (windows 10) I raised internal IT tickets as I suspected it was linked to access credentials. Service desk uninstalled the Anaconda from ProgramData and make me reinstall in local folder.  Then I saw the root environment changed to (writable), previously it was (readable)

![image](https://user-images.githubusercontent.com/37668921/38833676-da7714c4-418b-11e8-9104-38354ad47e9d.png)
